src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -139,6 +142,26 @@
     if ($exit) {
         exit($exit);
     }
+
+    // Mutate TestsCase code
+    $alteredCode = file_get_contents($alteredFile = implode(DIRECTORY_SEPARATOR, array('.', 'src', 'Framework', 'TestCase.php')));
```

Comment:
no need for implode here, `/` works on all OSes, isn't it?

Reply:
fixed